include/taskolib/Context.h

```diff
@@ -42,9 +42,14 @@ namespace task {
 /**
  * A VariableValue is a variant over several data types (long long, double, std::string).
  *
+ * The char pointer will be converted to a std::string when the variable is imported
+ * into a Step by copying the contents. That can be a long time after assignment.
+ * Be careful with the lifetime of the pointed to string; it must outlife the Context.
+ * Normally you should prefer to hand over std::string.
+ *
  * Names are associated with these values via a map in the Context type.
  */
-using VariableValue = std::variant<long long, double, std::string>;
+using VariableValue = std::variant<long long, double, std::string, char const*, bool>;
```

Comment:
Why do we add here beside `bool` also `char const*`? We have, as you already noticed in the conversation, the type `std::string` that already represents a character array.